cmd/k8s-operator/operator.go

```diff
@@ -150,6 +151,13 @@ func initTSNet(zlog *zap.SugaredLogger) (*tsnet.Server, *tailscale.Client) {
 		Hostname: hostname,
 		Logf:     zlog.Named("tailscaled").Debugf,
 	}
+	if p := os.Getenv("TAILSCALED_PORT"); p != "" {
```

Comment:
`TAILSCALED_PORT` seems weird for a `tsnet` application. Looking at other env vars, it seems like we have a lot of consistency about generally using `TS_` as a prefix, and that also seems more fitting:

```suggestion
	if p := os.Getenv("TS_PORT"); p != "" {
```

Reply:
Thanks @tomhjp I've changed this to `TS_PORT`